Formula/g/gitversion@5.rb

```diff
@@ -0,0 +1,45 @@
+class GitversionAT5 < Formula
+  desc "Easy semantic versioning for projects using Git"
+  homepage "https://gitversion.net"
+
+  url "https://github.com/GitTools/GitVersion/archive/refs/tags/5.12.0.tar.gz"
+  sha256 "fe2ecbd2d63a4458f19eb9f0ee6853b5041e8b2f6d7c75b0fa606be2d1a81476"
+  license "MIT"
+
+  keg_only :versioned_formula
+  depends_on "dotnet@6"
```

Comment:
`dotnet@6` will be deprecated on 2024-11-12 in alignment with official Microsoft support[^1]

Any dependents will also be deprecated on the same date, so this formula will only have a lifespan of 1.5 months. Thus this doesn't really meet our requirements for versioned formulae.

[^1]: https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#lifecycle

Reply:
@cho-m ok got it

I will close this PR then